spek-ide-plugin/intellij-common/build.gradle

```diff
@@ -0,0 +1,11 @@
+apply from: "$rootDir/gradle/common/dependencies.gradle"
+apply from: "$rootDir/gradle/common/idea-plugin.gradle"
+
+intellij {
+    version = '2016.3'
+    plugins = ['org.jetbrains.kotlin:1.2.0-rc-39-IJ2016.3-5@EAP-1.2']
```

Comment:
The plugin version does not match the project version. Is it OK?

Reply:
Yes, trusting JetBrains not to break backward compatibility. But I'll update it just in case. 👍